include/onnxruntime/core/graph/graph.h

```diff
@@ -1141,8 +1142,22 @@ class Graph {
   */
   Status InlineFunction(Node& node);
 
+  /**
+  Directly insert the nodes in the function proto provided into the graph.
+  The function converts Constant nodes into the initializers in the graph.
+  It then creates a node in the graph for each of the function nodes.
+  All of the names are expected to be specialized, and, therefore unique.
+  See function_utils::Specialize().
+
+  The Graph needs to be Resolve()d after this call.
+  @param func_to_inline
+  @returns Status indicating success or providing an error message.
+  */
+
+  Status FunctionToGraph(const ONNX_NAMESPACE::FunctionProto& func_to_inline);
```

Comment:
Should this be named InlineFunction based on the description? The current name suggests that it converts a function to a Graph.

Reply:
This function takes a `FunctionProto` and inlines it into `this` Graph.

Reply:
Then may be we should name this to InlineFunctionProto.